core/src/test/scala/unit/kafka/tools/StorageToolTest.scala

```diff
@@ -433,5 +433,91 @@ Found problem:
       contains("Formatting dynamic metadata voter directory %s".format(availableDirs.head)),
       "Failed to find content in output: " + stream.toString())
   }
-}
 
+  private def runVersionMappingCommand(
+    stream: ByteArrayOutputStream,
+    properties: Properties,
+    releaseVersion: String
+  ): Int = {
+    val tempDir = TestUtils.tempDir()
+    try {
+      // Write the properties to a temporary file
+      val configPathString = new File(tempDir.getAbsolutePath, "version-mapping.props").toString
```

Comment:
Do we need a properties file for this command?

Reply:
I got an exception without the props file for certain configs not being present so this is what worked, I'm not sure if there's a better way

Reply:
Hmm... it appears like the uuid command doesn't need it? Is it the case that you could remove line 262 and it would work?

Reply:
Ah yes my bad I didn't realize I added the config arg method for the version mapping parser when I was making the code modular, thanks for the catch! Updated the tests as well